client/src/modules/enterprises/enterprises.html

```diff
@@ -217,6 +217,18 @@
             <i class="fa fa-cogs"></i> <span translate>ENTERPRISE.SETTINGS.TITLE</span>
           </div>
           <div class="panel-body">
+              <div class="form-group" ng-class="{ 'has-error' : EnterpriseForm.$submitted && EnterpriseForm.month_average_consumption.$invalid }">
+                <label class="control-label" translate>
+                  FORM.LABELS.DEFINE_MONTH_AVERAGE_CONSUMPTION
+                </label>
+                <input type="number" class="form-control" autocomplete="off"
+                  name="month_average_consumption" required ng-model="EnterpriseCtrl.enterprise.settings.month_average_consumption"
```

Comment:
Does it need to be required?  Or do you set it to a default value in the enterprise controller?  There are some institutions that won't use stock management, so we shouldn't force them to set this variable unnecessarily.